build.sh

```diff
@@ -158,8 +158,12 @@ if [ ${BUILD_MACOS} -gt 0 ]; then
     popd
 fi
 
+source lib/variables.sh
+VERSION="$VIPS_VERSION-$(git rev-parse --short HEAD)"
 
+mvn ${MAVEN_ARGS} -DnewVersion=$VERSION versions:set
 mvn ${MAVEN_ARGS} -DskipTests clean package
+mvn ${MAVEN_ARGS} versions:revert
```

Comment:
or -newVersion=$VERSION-SNAPSHOT?

Reply:
It's not exactly the same. `versions:set` will create a `pom.xml.backup` file and use it when calling `versions:revert`.

Reply:
ah yes, that's true... let's keep it like this then